hardware/tv-hat/README.md

```diff
@@ -0,0 +1,58 @@
+## Getting started with the Raspberry Pi TV HAT
+
+The TV HAT allows you to receive and decode digital television streams on your Raspberry Pi through its on-board DVB-T2 tuner. Then you can watch these streams on any computer connected to the same network as the Pi.
+
+The software we recommend to decode the streams (known as multiplexes, or muxes for short) and view content is called TVHeadend, and instructions for setting it up are below. The TV HAT can decode one mux at a time, and each mux can contain several channels to choose from. Content can either be viewed on the Raspberry Pi to which the TV-HAT is connected, or sent to another device on the same network.
+
+**You will need:**
+* A TV aerial
+* A Raspberry Pi TV HAT with its stand-offs, screws, and aerial adaptor
+* A Raspberry Pi that is connected to the internet (plus a mouse, keyboard, and display, if
+you are not accessing it remotely)
+* Another computer connected to the same network
+
+### Setup instructions
+
+**On your Raspberry Pi:**
+
+* Connect the aerial adaptor to the TV HAT. With the adaptor pointing away from the USB ports, press the HAT gently down over the Raspberry Pi’s GPIO pins. Place the spacers at two or three of the corners of the HAT, and tighten the screws through the mounting
+holes to hold them in place.
+* Connect the TV HAT’s aerial adaptor to the cable from your TV aerial.
+* Set up the Raspberry Pi with the newest version of the Raspbian operating system, which you can download from [rpf.io/raspbian](rpf.io/raspbian)
+ * If you don’t know how to do this, follow our guide at [rpf.io/startwithpi](rpf.io/startwithpi)
+* Start up your Pi, open a terminal window, and run the following two commands to install the TVHeadend software:
+```
+sudo apt-get update
+sudo apt-get install tvheadend
+```
+  * If you don’t know how to do this, follow our guide at [rpf.io/usingterminal](rpf.io/usingterminal)
+* During the TVHeadend installation, you will be asked to choose an administrator account name and password. You’ll need these later, so make sure to pick something you can remember.
+
+**In a web browser on a different computer:**
+
+* Type the following into the address bar: `http://raspberrypi.local:9981/extjs.html`
+* This should connect to TVHeadend running on the Raspberry Pi.
+  * If the address above doesn't work, you’ll need to find out the IP address of the Pi. Open a terminal window on your Pi, and run the command hostname -I
+  * You’ll see the IP address in two formats: a string of four numbers separated by dots, then a space, then a long string of numbers and letters separated by colons.
```

Comment:
Is the second part an IPv6 address? Presumably that won't be present if the network the Pi is connected to isn't using IPv6?